### PR TITLE
chore: Improvise GitHub Issue Templates

### PR DESCRIPTION
**Pull request by kushthedude**

### CODE REVIEW

File: .github/ISSUE_TEMPLATE/bug_report.md [hunk 1]
```diff
@@ -6,13 +6,13 @@ labels: 'kind/bug'
 assignees: ''
 ---
 **Description**
-A brief description of the issue.
+<!-- A brief description of the issue. -->
```
Comment (leecalcote) [inline .github/ISSUE_TEMPLATE/bug_report.md:9]: Good. 👍 These instructions are for the issue-creator and can be hidden.

File: .github/ISSUE_TEMPLATE/bug_report.md [hunk 2]
```diff
@@ -21,22 +21,24 @@ Add screenshots, if applicable, to help explain your problem.
...
 4. See error
 
 [Optional] **Additional Context**
-Add any other context about the problem here.
+<!-- Add any other context about the problem here. -->
 
 ---
 **Contributor Resources**
-The `layer5` repo contains two websites. The current generation and the next-generation of the layer5.io site.
+<!-- The `layer5` repo contains two websites. The current generation and the next-generation of the layer5.io site.
```
Comment (leecalcote) [inline .github/ISSUE_TEMPLATE/bug_report.md:38]: These instructions are for the contributor as the time of picking up the issue and working on it, so they will need to be able to see these as they won't know what's going on and the fact that there are two sites in this repo.

Comment (kushthedude) [reply .github/ISSUE_TEMPLATE/bug_report.md:38]: I didn't know it was for the purpose of the contributor :+1:

Comment (kushthedude) [reply .github/ISSUE_TEMPLATE/bug_report.md]: Reverted

**Pull request merged**